budgetkey_data_pipelines/pipelines/entities/moital/pipeline-spec.yaml

```diff
@@ -0,0 +1,14 @@
+moital_service_providers:
+  pipeline:
+    - run: add_metadata
+      parameters:
+        name: moital_service_providers
```

Comment:
So, we have two datasets:
Service contractors: http://apps.moital.gov.il/WebServicesHandlers/ServiceContractors.aspx
Manpower contractors: http://apps.moital.gov.il/WebServicesHandlers/ManpowerContractors.aspx

we should keep names consistent - this dataset is called `moital_service_providers` but in fact scrapes `ManpowerContractors.aspx`...